src/types/contracts/common.ts

```diff
@@ -49,11 +49,11 @@ export type EndowmentStatus = {
   closed: 3;
 };
 export type EndowmentType = "charity" | "normal";
-export type CapitalizedEndowmentType = Capitalize<EndowmentType>;
+// export type CapitalizedEndowmentType = Capitalize<EndowmentType>;
 export type EndowmentStatusText = keyof EndowmentStatus;
 export type EndowmentStatusNum = EndowmentStatus[EndowmentStatusText];
 export type EndowmentStatusStrNum = `${EndowmentStatusNum}`;
-export type EndowmentTier = "Level1" | "Level2" | "Level3";
```

Comment:
> There's also a new field called `endow_tier` that will replace the `tier` once v2.0 of the smart contracts is live in production... The `tiers` param's name remains unchanged and the required check is done in the Lambda itself.

@jp-mariano will this field be renamed once contracts are live?

Reply:
Yes, this was necessary to update the field's type from `String` to `Integer` without breaking the live app. Right now, the check is applied to the Lambda directly so that it knows if `"Level3"` or `3` is passed then it will use the correct field name.